drainer/syncer.go

```diff
@@ -492,7 +492,7 @@ ForLoop:
 					continue
 				}
 			} else if !ignore && s.cfg.DestDBType == "oracle" {
-				if _, ok := stmt.(*ast.TruncateTableStmt); !ok {
+				if _, ok := stmt.(*ast.TruncateTableStmt); !ok && s.cfg.SyncDDL {
```

Comment:
I'm a bit confuse why we only check `s.cfg.SyncDDL` in such inner block. Is it bettern to check and skip all DDL event at L433. @lichunzhu

Reply:
Because we don't `continue` here but just set `shouldSkip = true` to make sure this table's schema can be updated in time. Maybe we should return an error in oracle dsyncer's loader, but I think return an error here is also okay.

Reply:
I mean we should check `!s.cfg.SyncDDL` to L433 instead of check here.

Reply:
SGTM, but we need to sync this item at L433 too.